src/praisonai-agents/test_architectural_fixes.py

```
@@ -1,161 +1,161 @@
+    task.retry_count = 0
+    
+    # Create a mock task output
+    output = TaskOutput()
+    output.raw = "test output"
+    
+    # Test the callback path - should no longer process guardrails
+    try:
+        import asyncio
+        asyncio.run(task.execute_callback(output))
+        print("✓ Guardrail processing moved from callback to execution path")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)